[2022-11-02 14:36:35,253] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:35:52.526196+00:00 [queued]>
[2022-11-02 14:36:35,289] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:35:52.526196+00:00 [queued]>
[2022-11-02 14:36:35,291] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-02 14:36:35,297] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-11-02 14:36:35,299] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-02 14:36:36,305] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_xetra_etl> on 2022-11-02 14:35:52.526196+00:00
[2022-11-02 14:36:37,618] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xetra_DAG', 'run_xetra_etl', 'manual__2022-11-02T14:35:52.526196+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/xetra_DAG.py', '--cfg-path', '/tmp/tmpvctjg2z_', '--error-file', '/tmp/tmpwqvm4v0_']
[2022-11-02 14:36:37,621] {standard_task_runner.py:80} INFO - Job 138: Subtask run_xetra_etl
[2022-11-02 14:36:37,609] {standard_task_runner.py:52} INFO - Started process 175 to run task
[2022-11-02 14:36:42,411] {task_command.py:369} INFO - Running <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:35:52.526196+00:00 [running]> on host 0163e0790208
[2022-11-02 14:37:05,635] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xetra_DAG
AIRFLOW_CTX_TASK_ID=run_xetra_etl
AIRFLOW_CTX_EXECUTION_DATE=2022-11-02T14:35:52.526196+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-02T14:35:52.526196+00:00
[2022-11-02 14:37:05,641] {logging_mixin.py:115} INFO - /opt/***/meta_file.csv
[2022-11-02 14:38:04,138] {python.py:173} INFO - Done. Returned value was: None
[2022-11-02 14:38:04,206] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xetra_DAG, task_id=run_xetra_etl, execution_date=20221102T143552, start_date=20221102T143635, end_date=20221102T143804
[2022-11-02 14:38:05,328] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-02 14:38:11,555] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
